cpp/density/RDF.h

```diff
@@ -16,8 +16,16 @@ namespace freud { namespace density {
 class RDF : public locality::BondHistogramCompute
 {
 public:
+    //! Enum for each normalization mode
+    enum NormalizationMode
```

Comment:
It's up to you whether to do it in this PR or to fix all instances of this in a follow-up PR, but I would recommend moving all uses of `enum` in freud to `enum class`. Scoped enums are definitely preferable (although they require very slightly more hoops to jump through with Cython until we can update to Cython 3.0) but given that all our enums are unscoped right now I don't mind adding this one like this for now then changing them all later. Up to you.

Reply:
I agree with the enum classes suggestion, but I'll wait until the cython 3.0 release to take a PR out to handle it. If you'd like to make an issue so we can track it, I would appreciate it. In the meantime, this particular enum is wrapped by the RDF class, so it doesn't leak outside `freud::density::RDF`.

Reply:
Done: https://github.com/glotzerlab/freud/issues/1025.